### PR TITLE
Issue 961 add find for multiple queries

### PR DESCRIPTION
**Pull request by akrystian**

### CODE REVIEW

File: javers-core/src/test/groovy/org/javers/core/JaversRepositoryShadowE2ETest.groovy [hunk 1]
```diff
@@ -669,6 +664,52 @@ class JaversRepositoryShadowE2ETest extends JaversRepositoryE2ETest {
...
+                (1..5).collect { new DummyAddress(city: "London${it}") }
+                        + (1..5).collect { new DummyPoint(it, 2) }
+                        + new DummyPoint(6, 2), //noise
+                (1..5).collect { new SnapshotEntity(id: 1, valueObjectRef: new DummyAddress(city: "London${it}")) }
+                        + (1..5).collect { new SnapshotEntity(id: 2, valueObjectRef: new DummyAddress(city: "Paris${it}")) }
+                        + new SnapshotEntity(id: 3, valueObjectRef: new DummyAddress(city: "London1")) //noise
+        ]
+        query << [
+                [byInstanceId(1, SnapshotEntity).limit(3).build(), byInstanceId(2, SnapshotEntity).limit(3).build()],
+                [QueryBuilder.byClass(DummyAddress).limit(3).build(), QueryBuilder.byClass(DummyPoint).limit(3).build()],
```
Comment (akrystian) [inline javers-core/src/test/groovy/org/javers/core/JaversRepositoryShadowE2ETest.groovy:699]: I have problem with this case. It doesn't collect any shadows and I don't know why.

File: javers-core/src/main/java/org/javers/core/JaversCore.java [hunk 2]
```diff
@@ -187,6 +188,12 @@ public Diff initial(Object newDomainObject) {
         return (List)queryRunner.queryForShadows(query);
     }
 
+    @Override
+    public <T> List<Shadow<T>> findShadows(List<JqlQuery> queries) {
```
Comment (bartoszwalacik) [inline javers-core/src/main/java/org/javers/core/JaversCore.java:192]: List of queries to perform one query is not ok, please add multiple ID support to  proper Filter

Comment (akrystian) [reply javers-core/src/main/java/org/javers/core/JaversCore.java]: I removed it. But one test from core don't pass.

Comment (bartoszwalacik) [reply javers-core/src/main/java/org/javers/core/JaversCore.java]: what now?

**Comment by bartoszwalacik** [comment]
@akrystian this branch has obvious compilation errros, 
you didn't implemented changes in MongoRepository and SqlRepository, what's going on here?

**Pull request closed without merging**